app/views/pages/dashboard.html.erb

```diff
@@ -1,40 +1,57 @@
 <div class="container">
-  <h1>
-    Hello, <%= current_user.email_or_first_name %>
-  </h1>
-
-  <div>
-    <h2>Booked Spaceships</h2>
-    <div class="row">
-      <% current_user.booked_spaceships.each do |spaceship| %>
-        <div class="col-sm-6 col-md-4 col-lg-3 col-xs-12">
-          <div class="card">
-            <p>Model: <%= spaceship.model %></p>
-            <h4>Ship details :</h4>
-            <p>
-              This ship has a capacity of <%= spaceship.capacity %>.
-              Its location is <%= spaceship.location %>.
-              The captain of the ship is <%= spaceship.captain_name %>.
-            </p>
-          </div>
+  <br>
+  <br>
+  <br>
+  <br>
+
+  <br>
+  <br>
+  <br>
```

Comment:
I've added line breaks already so I am not sure if this would not overwrite something.

Reply:
I just wanted to see the changes i was making and forgot to take them out. these can be deleted